packages/dev/core/src/Cameras/arcRotateCamera.ts

```diff
@@ -1087,15 +1087,14 @@ export class ArcRotateCamera extends TargetCamera {
     public setTarget(target: TransformNode | Vector3, toBoundingCenter = false, allowSamePosition = false, cloneAlphaBetaRadius = false): void {
         cloneAlphaBetaRadius = this.overrideCloneAlphaBetaRadius ?? cloneAlphaBetaRadius;
 
-        const canTakeHostPath = (!toBoundingCenter || (toBoundingCenter && (<any>target).getBoundingInfo)) && (target as TransformNode).computeWorldMatrix;
-        if (canTakeHostPath) {
-            if (toBoundingCenter) {
+        if ((target as TransformNode).computeWorldMatrix) {
+            if (toBoundingCenter && (<any>target).getBoundingInfo) {
                 this._targetBoundingCenter = (<any>target).getBoundingInfo().boundingBox.centerWorld.clone();
             } else {
                 this._targetBoundingCenter = null;

```

Comment:
should this be simply the transformnode position ?

Reply:
Yes, the `this._target = this._getTargetPosition();` call on line 1098 will use the transform node position for the target.